sdk/docs/pages/dapp-kit/wallet-hooks/useWallets.mdx

```diff
@@ -0,0 +1,41 @@
+import { UseWalletsExample } from '../../../examples/wallet-hooks';
+
+# `useWallets`
+
+The `useWallets` hook returns an array of wallets that are available to the user. The wallets are
+sorted by their priority, with the highest priority wallet being the first in the array.
+
+```ts
+import { useWallets } from '@mysten/dapp-kit';
+
+function MyComponent() {
+	const wallets = useWallets();
+
+	return (
+		<div>
+			<h2>Installed wallets</h2>
+			{wallets.length === 0 && <div>No wallets installed</div>}
+			<ul>
+				{wallets.map((wallet) => (
+					<li key={wallet.name}>{wallet.name}</li>
+				))}
+			</ul>
+		</div>
+	);
+}
+```
+
+## Example
+
+<UseWalletsExample />
+
+## Wallet properties
```

Comment:
Ideally there'd be good wallet standard docs to point to since this is really just a wallet standard object.